src/apps/lwaftr/binding_table.lua

```diff
@@ -143,10 +143,31 @@ local BindingTable = {}
 function BindingTable.new(psid_map, br_addresses, softwires)
    local ret = {
       psid_map = assert(psid_map),
-      br_addresses = assert(br_addresses),
-      softwires = assert(softwires)
+      softwires = assert(softwires),
+      br_addresses = cltable.new({key_type = ffi.typeof('uint8_t[16]')})
```

Comment:
I think the reason why the softwires were layout this way was to compact them as much as possible, so more softwires fit in cache. That will increase the number of cache hits.

It might be the case that this change have an impact in performance, but according to the benchmarks we run so far it seems performance is as good as the `lwaftr` branch.

As for me, I think we can go ahead with merging the PR since in the mid-term these changes will only live in the `lwaftr-kristian` branch, where we can test it more thoroughly.